src/chronometer.js

```diff
@@ -1,33 +1,62 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+    }
+   currentTime = 0
+    intervalId = null
+
+
+  start(printTimeCallback){
+    let currentTime = 0;
 
-  start(printTimeCallback) {
-    // ... your code goes here
+function start(printTimeCallback) {
+  let currentTime = 0;
+
+function start(printTimeCallback) {
+    currentTime = 0;
+    let iidInterval = setInterval(() => {
+        currentTime++;
+        printTimeCallback(currentTime);
+    }, 3000);
+}
+}
   }
 
   getMinutes() {
-    // ... your code goes here
+   return Number(Math.ceil(this.currentTime/1000))
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor((this.currentTime % 1000))
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+      let Value = value.toString();
+      if (Value.length === 1) {
+          Value = "0" + Value;
+      }
+      return Value;
   }
 
+      
+
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId)
```

Comment:
Debes inicializar tu propiedad `this.intervalId` y guardar el id de tu intervalo para poder utilizarlo en `clearInterval`.